.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+## Proposed changes
+
+Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
+
+<!-- Github Tip: adding the text 'Fixes #<issue>' or 'Closes #<issue>' will automatically close the mentioned issue. -->
+
+## PR Checklist
+
+<!-- _Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._ -->
+
+- [ ] I have read the [CONTRIBUTING](https://github.com/coreruleset/coreruleset/blob/v4.0/dev/CONTRIBUTING.md) documentation.
+- [ ] I have tested the changes locally.
```

Comment:
This doesn't make sense for documentation.

Reply:
Fixed in [f8a8b1d](https://github.com/coreruleset/documentation/pull/132/commits/f8a8b1d136f7c5ab726e33c8fab633d945b6dbfd).